src/custom-element.js

```diff
@@ -549,7 +552,11 @@ export function createAmpElementProto(win, name, opt_implementationClass) {
         // Few top elements will also be pre-initialized with a loading
         // element.
         this.getVsync_().mutate(() => {
-          this.prepareLoading_();
+          // Repeat "loading enabled" check because it could have changed while
+          // waiting for vsync.
+          if (this.isLoadingEnabled_()) {
+            this.prepareLoading_();
+          }
```

Comment:
`prepareLoading_()` 's caller must ensure that loading can actually be shown. I add another check in case `isLoadingEnabled_()` value changes during the wait for vsync